### ISSUE TITLE
[packagers] Docker throws NPE with SINGLE_JAR distribution

### ISSUE THREAD

**Issue by aalmiray** [jreleaser/jreleaser#1258]
### Task List

- [x] Steps to reproduce provided
- [x] Stacktrace (if present) provided
- [x] Example that reproduces the problem (link to git repository is ideal)
- [x] Full description of the issue provided (see below)

### Steps to Reproduce

1. Clone https://github.com/jreleaser/helloworld-java-jar
2. Modify the distribution configuration by adding `docker`
```
distributions:
  helloworld:
    type: SINGLE_JAR
    artifacts:
      - path: target/{{distributionName}}-{{projectVersion}}.jar
    docker:
      active: ALWAYS
```
3. Invoke `jreleaser package`

### Expected Behaviour

Successful docker image created.

### Actual Behaviour

NPE occurs during validation

```
java.lang.NumberFormatException: null
	at java.base/java.lang.Integer.parseInt(Integer.java:614)
	at java.base/java.lang.Integer.parseInt(Integer.java:770)
	at org.jreleaser.model.internal.validation.packagers.DockerPackagerValidator.validateBaseImage(DockerPackagerValidator.java:284)
	at org.jreleaser.model.internal.validation.packagers.DockerPackagerValidator.validateDocker(DockerPackagerValidator.java:124)
	at org.jreleaser.model.internal.validation.distributions.DistributionsValidator.validateDistribution(DistributionsValidator.java:209)
	at org.jreleaser.model.internal.validation.distributions.DistributionsValidator.validateDistributions(DistributionsValidator.java:84)
	at org.jreleaser.model.internal.JReleaserModelValidator.validateModel(JReleaserModelValidator.java:79)
	at org.jreleaser.model.internal.JReleaserModelValidator.validate(JReleaserModelValidator.java:57)
	at org.jreleaser.model.internal.JReleaserContext.validateModel(JReleaserContext.java:479)
	at org.jreleaser.engine.context.ModelValidator.validate(ModelValidator.java:47)
	at org.jreleaser.workflow.WorkflowImpl.<init>(WorkflowImpl.java:45)
	at org.jreleaser.workflow.Workflows.packageRelease(Workflows.java:120)
	at org.jreleaser.cli.Package.doExecute(Package.java:36)
	at org.jreleaser.cli.AbstractModelCommand.execute(AbstractModelCommand.java:89)
	at org.jreleaser.cli.AbstractCommand.call(AbstractCommand.java:52)
	at org.jreleaser.cli.AbstractModelCommand.call(AbstractModelCommand.java:53)
	at org.jreleaser.cli.AbstractCommand.call(AbstractCommand.java:37)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2041)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
	at picocli.CommandLine.execute(CommandLine.java:2170)
	at org.jreleaser.cli.Main.execute(Main.java:98)
	at org.jreleaser.cli.Main.run(Main.java:81)
	at org.jreleaser.cli.Main.main(Main.java:70)
```

### Environment Information

- **Operating System**: N/A
- **JReleaser Version:** 1.5.0
- **JDK Version:** N/A

**Comment by aalmiray** [comment]
Either `project.java.version` or `distributions.helloworld.java.version` must be defined. The error is caused when neither isn't.

**Comment by aalmiray** [comment]
🎉 This issue has been resolved in `v1.5.1` ([Release Notes](https://github.com/jreleaser/jreleaser/releases/tag/v1.5.1))
